src/components/learner-credit-management/data/hooks.js

```diff
@@ -63,18 +63,15 @@ const applySortByToOptions = (sortBy, options) => {
 };
 
 const applyFiltersToOptions = (filters, options) => {
-  const userSearchQuery = filters?.find(filter => filter.id === 'userEmail')?.value;
-  const courseTitleSearchQuery = filters?.find(filter => filter.id === 'courseTitle')?.value;
   const courseProductLineSearchQuery = filters?.find(filter => filter.id === 'courseProductLine')?.value;
-  if (userSearchQuery) {
-    Object.assign(options, { search: userSearchQuery });
-  }
-  if (courseTitleSearchQuery) {
-    Object.assign(options, { searchCourse: courseTitleSearchQuery });
-  }
+  const searchQuery = filters?.find(filter => filter.id.toLowerCase() === 'enrollment details')?.value;
```

Comment:
[curious/clarification] Do you happen to know where this filter id of "enrollment details" (with the space) is coming from? My understanding was that the `filter.id` was based on the `accessor` for the column (e.g., like `courseProductLine` above). But the accessor for the "Enrollment details" column is an object (first time seeing this).

Reply:
Yeah I was surprised as well. It appears that if the accessor is a function (which in this case, it is), it defaults the `filter.id` to the `Header` of the column.